.github/workflows/pr.yml

```diff
@@ -9,7 +9,22 @@ jobs:
     uses: the-guild-org/shared-config/.github/workflows/algolia-integrity.yml@main
     with:
       domain: https://www.the-guild.dev/graphql/sofa-api/
-      source: 'SOFA'
-      setupEnvWorkingDirectory: './website'
+      source: "SOFA"
+      setupEnvWorkingDirectory: "./website"
+    secrets:
+      githubToken: ${{ secrets.GITHUB_TOKEN }}
+
+  release:
+    uses: the-guild-org/shared-config/.github/workflows/release-snapshot.yml@main
+    with:
+      npmTag: alpha
+      buildScript: build
+      nodeVersion: 18
```

Comment:
should be 16? the `release.yaml` is using 16... let's make sure it matches the other pipelines in SOFA repo